apps/dashboard/src/lib/components/Org/SideBar.svelte

```diff
@@ -62,6 +39,44 @@
   const openModal = () => {
     $isUpgradeModalOpen = true;
   };
+
+	$: {
+		menuItems = [
+	    {
+	      path: '',
+	      label: 'Dashboard',
+	      show: true
+	    },
+	    // {
+	    //   path: '/quiz',
+	    //   label: 'Quizzes'
+	    // },
+	    {
+	      path: '/courses',
+	      label: 'Courses',
+	      show: true
+	    },
+	    {
+	      path: '/community',
+	      label: 'Community',
+	      show: true
+	    },
+	    // {
+	    //   path: '/site',
+	    //   label: 'Site settings',
+	    // },
+	    {
+	      path: '/audience',
+	      label: 'Audience',
+	      show: true
+	    },
+	    {
+	      path: '/setup',
+	      label: 'Setup',
+	      show: $isOrgAdmin
```

Comment:
yea, making the menu items reactive actually makes sense since the isOrgAdmin value could change and we want the menu to dynamically render accordingly